test/decl/protocol/special/coding/Inputs/class_codable_simple_multi1.swift

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: objc_interop
```

Comment:
These require the import of Foundation, which necessitates the inclusion of `objc_interop`, no?